jekyll/_cci2/configuration-cookbook.md

```diff
@@ -626,6 +626,7 @@ In the above configuration, we:
 - Lastly, we call the `setup` job defined above as a part of our `workflow`
 
 Note: You can only use one workflow per `config.yml` when using CircleCI's dynamic configuration feature
+Note: You can't call CircleCI's dynamic configuration feature several times in the same pipeline. It means you can't make the another workflow with `setup: true` by the CircleCI's dynamic configuration feature.
```

Comment:
```suggestion
Note: You are unable to call CircleCI's dynamic configuration feature several times in the same pipeline. This means that you cannot make the another workflow with `setup: true` by the CircleCI's dynamic configuration feature.
```

Reply:
> This means that you cannot make the another workflow with `setup: true` by the CircleCI's dynamic configuration feature.

Is still unclear to me.
maybe ...

This means that you cannot make a second workflow with `setup: true` within the same `.circleci/config.yml` file.